iterative_verifier.go

```diff
@@ -337,7 +337,7 @@ func (v *IterativeVerifier) reverifyUntilStoreIsSmallEnough(maxIterations int) e
 		before := v.reverifyStore.RowCount
 		start := time.Now()
 
-		_, err := v.verifyStore("reverification_before_cutover", []MetricTag{{"iteration", string(iteration)}})
+		_, err := v.verifyStore("reverification_before_cutover", []MetricTag{{"iteration", fmt.Sprint(iteration)}})
```

Comment:
Also is this change made purposefully ?

Reply:
Yes, the compiler was complaining and wouldnt compile until this was fixed

string(int) creates a rune with the ascii / UTF-8 int code